x-pack/plugins/reporting/server/routes/common/authorized_user_pre_routing.ts

```diff
@@ -30,15 +30,15 @@ export const authorizedUserPreRouting = <P, Q, B>(
   reporting: ReportingCore,
   handler: RequestHandlerUser<P, Q, B>
 ): RequestHandler<P, Q, B, ReportingRequestHandlerContext, RouteMethod> => {
-  const { logger, security, docLinks } = reporting.getPluginSetupDeps();
+  const { logger, security: securitySetup, docLinks } = reporting.getPluginSetupDeps(); // ReportingInternalSetup.security?: SecurityPluginSetup | undefined
 
   return async (context, req, res) => {
-    const { security: securityStart } = await reporting.getPluginStartDeps();
+    const { security: securityService } = await reporting.getPluginStartDeps();
```

Comment:
If we use `securityService` for naming the field in `ReportingInternalStart`, this becomes:
```suggestion
    const { securityService } = await reporting.getPluginStartDeps();
```